### ISSUE TITLE
Lab 2, βήμα #14

### ISSUE THREAD

**Issue by giannispic** [slp-ntua/patreco-lab-support-19-20#19]
Χωρίς τη χρήση pack padded sequence στη μέθοδο forward, το δίκτυο κάνει train (train loss πρακτικά μηδενίζεται), όμως το validation loss μειώνεται ελάχιστα και στη συνέχεια αυξάνεται με τη πάροδο των εποχών, δηλαδή γίνεται overfitting πολύ γρήγορα. Με χρήση pack padded sequence το πρόβλημα αυτό δεν υφίσταται. Αυτό είναι αναμενόμενο αποτέλεσμα??? (μαζί με τη μείωση του απαιτούμενου χρόνου για εκπαίδευση)

**Comment by georgepar** [comment]
αυτό έχει να κάνει με κλασικό bug όπου συνυπολογίζετε τα padded elements στο loss σας. δείτε τι κάνει το ορισμα ignore_index στο crossentropyloss η φίλτραρετε το με το χέρι κατά τον υπολογισμό
